app/models/rules_factory.rb

```diff
@@ -4,10 +4,12 @@ def initialize(auction)
   end
 
   def create
+    eligibility = EligibilityFactory.new(auction).create
```

Comment:
if it were me, I might stick this up in `initialize` to move it as far outside of this obj as possible. The next step would be to pass it in as an arg to `initialize`, but like @harrisj says, that can happy later